### ISSUE TITLE
How to update Gitea to the latest version?

### ISSUE THREAD

**Issue by SimonLab** [dwyl/gitea-server#6]
Gitea is updated frequently, https://github.com/go-gitea/gitea/tags
What is the process to update the version deployed on Fly?

Does having the `latest` version define in `fly.toml` enough? (I think yes)
https://github.com/dwyl/gitea-server/blob/9e33ad7756634b506813b6b3d35331e386cfae3f/fly.toml#L6-L7

How do we check the Gitea version on Fly to see if a deploy has updated the version?

- [ ] Update Readme to explain that Gitea version is updated automatically when running `flyctl deploy` as the Docker image get the `latest` Gitea` image version
- [ ] "We should investigate adding a "watcher" for the tag version on https://github.com/go-gitea/gitea/tags to trigger an update. Someone must have solved this problem already ... maybe we can use @dependabot" see https://github.com/dwyl/gitea-server/issues/6#issuecomment-1138311820

I'm currently experience slow query time when I push to a repository and was wondering if a new version of Gitea could fix that or if I need to investigate other paths

**Comment by SimonLab** [comment]
The version is displayed on the footer of Gitea:
![image](https://user-images.githubusercontent.com/6057298/170451685-64e5ba39-6063-4945-affc-1fcd6ff96e41.png)

We could try to redeploy to see if the version bump to 1.16.8

**Comment by nelsonic** [comment]
@SimonLab Yeah, I think it's enough to have `:latest` in the `fly.toml` file.
_however_ we don't currently have a way of _automatically_ deploying the server when a new release/tag is made for https://github.com/go-gitea/gitea/tags
![gitea-tag-v1 16 8](https://user-images.githubusercontent.com/194400/170453393-13a1d1e5-51b6-4dac-97e4-db3f0e1a58dd.png)


We are currently running `Gitea Version: 1.16.7` on https://gitea-server.fly.dev/
![gitea-version-1 16 7](https://user-images.githubusercontent.com/194400/170452909-4bb2968d-136b-42c8-a53b-15cb646f064f.png)

I just did a _manual_ deploy:

```sh
LOG_LEVEL=debug fly deploy --verbose
```

And it appears to have updated:
![gitea-version-1 16 8](https://user-images.githubusercontent.com/194400/170453187-b4157de3-8c7d-484f-b16f-07ca67c4c692.png)

We should investigate adding  a "watcher" for the tag version on https://github.com/go-gitea/gitea/tags to trigger an update. Someone _must_ have solved this problem already ... maybe we can use **`@dependabot`** 💭

**Comment by SimonLab** [comment]
Thanks for checking the update @nelsonic 
I've done a bit more test related to "slow" `push` (https://github.com/dwyl/gitea-server/issues/6#issue-1249164996), and use directly the `git push` command line from the terminal combine with `time`:
![image](https://user-images.githubusercontent.com/6057298/170457751-59a47775-75fa-45cc-91a1-82fba0d114ec.png)

We can see that the issue is linked directly to the gitea-server. We can test again later on if we upgrade the server "hardware" option on Fly

maybe some useful info from: https://github.com/go-gitea/gitea/issues/13706

**Comment by nelsonic** [comment]
@SimonLab it's very strange. perhaps we need a dedicated "`git push` round-trip time" issue here so that we can link to it and open a similar issue over on https://github.com/go-gitea/gitea/issues with _full_ context/explanation. 💭
